src/views/addProperty/addProperty.jsx

```diff
@@ -30,219 +31,198 @@ const validationSchema = Yup.object().shape({
   managers: Yup.array()
 });
 
-const formHandler = (data, context) => {
-  context.apiCall('post', '/properties', data, { success: 'Property Added!' });
-};
 
-export class AddProperty extends Component {
-  constructor(props) {
-    super(props);
+export const AddProperty = (props) => {
+  const [propertyManagers, setPropertyManagers] = useState([])
+  const [managerOptions, setManagerOptions] = useState([])
+  const [managerSelection, setManagerSelection] = useState([])
```

Comment:
This is actually the collection of results after a search, not the selected managers, so let's rename it to something like "filteredManagerOptions" and "setFilteredManagerOptions"